flask-connexion-rest/version_3/people.py

```diff
@@ -74,6 +74,7 @@ def create(person):
     :param person:  person to create in people structure
     :return:        201 on success, 406 on person exists
     """
+    global PEOPLE
```

Comment:
That shouldn't be necessary since we're not assigning to `PEOPLE` in this function (`PEOPLE = ...`). Instead, the existing `PEOPLE` dict gets updated: `PEOPLE[lname] = { ... `

Reply:
@dbader, here's the same behaviour reproduced on a micro level, it does not get updated:

```
In [9]: people = {'gupta': 'avishkar'}

In [10]: def update():
    ...:     people['sharma'] = 'avishkar'
    ...:

In [11]: people
Out[11]: {'gupta': 'avishkar'}
```